cves/kernel/CVE-2019-9162.yml

```diff
@@ -369,8 +398,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    The fix for this vulnerability was only an addition of code, and simply required
```

Comment:
Does this not add a new step to the current order of operations?

Reply:
Updated